docs/documentation/head/setup.md

```diff
@@ -42,3 +42,6 @@ Where MM is the major version, nn is the minor version and pp is the patch versi
 Versions for JDBC3 and lower can be found [here](https://jdbc.postgresql.org/download.html#others)
 This is a very brief outline of how to build the driver. Much more detailed information can be 
 found on the [github repo](https://github.com/pgjdbc/pgjdbc/blob/master/CONTRIBUTING.md)
+
+Even though the JDBC driver should be build with Gradle, for situations, where use of Gradle is not possible, e.g., when building pgjdbc for distributions, the pgjdbc provides a convenience source release artifact **-src.tar.gz**- a Maven based project. The Maven based project contains a minimal version of the JDBC driver with reduced feature and test set and is still validly buildable within the Maven build environment.
+Minimal version of the JDBC driver is available on ["Maven central repository"](https://repo1.maven.org/maven2/org/postgresql/postgresql/) or can be created with **gradlew -d :postgresql:sourceDistribution -Prelease**
```

Comment:
> for situations, where use of Gradle is not possible,

Then how can be created with `gradlew -d :postgresql:sourceDistribution -Prelease`?
If you can create the source distribution using Gradle, what stops you from using Gradle to build the driver?

Reply:
I can see a scenario where a user could create the source distribution and then build it somewhere else. I don't really have an issue with this.

Reply:
That probably could be a weird but possible scenario, but even then the way is redacted is not clear (at least for me).

Reply:
I have mentioned the creation of the Maven source project with Gradle because this is how the project is constructed and just after that uploaded to the Maven repository. I can imagine the situation when the user wants to generate its own Maven source release artifact and thus skip the maven central repository at all.
But it is true, that wording can be different here. I will rewrite this part.
Thanks for the note.